### PR TITLE
Add "Featured artist showcase beatmaps" wiki article

### PR DESCRIPTION
**Pull request by Nivalyx**
*((this took unexpectedly longer that i thought it would be whew ;~;))*

ok so i always had this idea of documenting all of the fa showcase beatmaps that have been made by the members of the Mappers' Guild in one page in a [Staff Log](http://osu.ppy.sh/wiki/Staff_Log)-kind of style, and... after putting together the many pieces of information that are scattered all around here is the said article at last, yeah 🧙

### CODE REVIEW

**Comment by mousesalt** [comment]
can you add one of my se-u-ra set ([Elif to Shiro Kura no Yoru -Called-](https://osu.ppy.sh/beatmapsets/1226466)). the set was actually made for the showcase but somehow it got missed to be on the news.

**Comment by Nivalyx** [comment]
@mousesalt true, added!

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
+# Featured artist showcase beatmaps
+
+Prior to the announcement of upcoming [featured artists](wiki/Featured_Artists), members of the [Mappers' Guild](/wiki/Mappers_Guild) who have either achieved Tier 1 (or higher) in [Guild rank](/wiki/Mappers_Guild#rewards) or been accepting certain [limited-time mystery quests](wiki/Mappers_Guild#quests) set by the Guild in advance are being invited every so often to create exclusive "showcase beatmaps" using songs and other materials from osu!'s featured artist backlog that are yet to be released. When the time to announce the featured artists in question are due, these beatmaps will be treated to a comprehensive coverage in the corresponding [news posts](https://osu.ppy.sh/home/news) and will occasionally be promoted in the [@osugame Twitter](https://twitter.com/osugame) as well.
+
+This article lists all the showcase beatmaps that have been created by the members of the Mappers' Guild over time.
+
+## 2017
+
+### November 2017
```
Comment (MilkyIQ) [inline wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md:9]: "2017" is redundant here, consider changing to just "November" (this goes for all others as well)

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+
+### August 2018
+
+- **[High Tea Music](https://osu.ppy.sh/home/news/2018-08-13-new-featured-artist-high-tea-music)** (2018-08-13)
+  - ![][o!s] [Andromedik - Invasion](https://osu.ppy.sh/beatmapsets/789544#osu/1656911) hosted by ![][flag_RU] [Mirash](https://osu.ppy.sh/users/2841009)
+  - ![][o!s] [Gourski x Himmes - Silence](https://osu.ppy.sh/beatmapsets/789905#osu/1657683) hosted by ![][flag_FR] [BOUYAAA](https://osu.ppy.sh/users/405449)
+  - ![][o!s] [James Portland - Sky](https://osu.ppy.sh/beatmapsets/789528#osu/1656874) hosted by ![][flag_SG] [handsome](https://osu.ppy.sh/users/2123087)
+  - ![][o!s] [Lexurus - Gravity](https://osu.ppy.sh/beatmapsets/789529#osu/1656879) hosted by ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994)
+  - ![][o!s] [Phonetic - Journey](https://osu.ppy.sh/beatmapsets/795432#osu/1670491) hosted by ![][flag_KR] [Left](https://osu.ppy.sh/users/5062061)
+  - ![][o!s] [Pulse - LP](https://osu.ppy.sh/beatmapsets/789374#osu/1656604) hosted by ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418)
```
Comment (MilkyIQ) [inline wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md:41]: these are mising the parentheses around the game mode icon. personally i think they look better without parentheses tho.

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+
+As the Guild itself was operating on a closed, application-based entry in its early days, access to upcoming featured artist contents were pften restricted only to a handful of mappers who were predominantly of the [osu!](wiki/Game_mode/osu!) game mode. However, [the Guild's shift to an open and automated entry system in early 2019](wiki/Mapping_and_Modding_Timeline#march.5) has gradually allowed more and more mappers to participate in the creation of showcase beatmaps for future featured artists over time regardless of their game mode given that they either have achieved Tier 1 (or higher) in [Guild rank](/wiki/Mappers_Guild#rewards) or been accepting certain [limited-time mystery quests](wiki/Mappers_Guild#quests) set by the Guild in advance.
+
+Starting from [MYUKKE.'s featured artist announcement](https://osu.ppy.sh/home/news/2020-12-05-new-featured-artist-myukke) in December 2020 onwards, new featured artist announcement posts have also began to feature short preview videos — often with these showcase beatmaps as their main headline — every now and then.
```
Comment (MilkyIQ) [inline wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md:324]:
```suggestion
Starting from [MYUKKE.'s featured artist announcement](https://osu.ppy.sh/home/news/2020-12-05-new-featured-artist-myukke) in December 2020 onwards, new featured artist announcement posts occasionally began to feature short preview videos, often with these showcase beatmaps acting as their main headline.
```

**Review by MilkyIQ — COMMENTED** [review]
Review for grammar and style

**Review by mousesalt — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
Please don't use the mode abbreviations (especially not `o!s`). Either convert to icons or use the full naming.

**Comment by Walavouchey** [comment]
They are icons and there are no occurrences of those abbreviations in actual text. View the rendered file.
image

**Comment by peppy** [comment]
That's fine, but the mode names still need to be renamed in the article.

**Comment by Walavouchey** [comment]
Are you talking about the icon alt-text? There are otherwise no mentions of the modes in the article.
image

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+
+[o!s]: /wiki/shared/mode/osu.png "osu!"
```
Comment (Walavouchey) [inline wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md:326]:
```suggestion
[o!s]: /wiki/shared/mode/osu.png "osu!standard"
```

**Comment by Nivalyx** [comment]
ah, okok, those alt texts are actually something that i took directly from this wiki article --> http://osu.ppy.sh/wiki/en/Game_modifier/ which i didn't realize have been outdated by now :x

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+  - ![][o!s] [Andromedik - Invasion](https://osu.ppy.sh/beatmapsets/789544#osu/1656911) hosted by ![][flag_RU] [Mirash](https://osu.ppy.sh/users/2841009)
+  - ![][o!s] [Gourski x Himmes - Silence](https://osu.ppy.sh/beatmapsets/789905#osu/1657683) hosted by ![][flag_FR] [BOUYAAA](https://osu.ppy.sh/users/405449)
+  - ![][o!s] [James Portland - Sky](https://osu.ppy.sh/beatmapsets/789528#osu/1656874) hosted by ![][flag_SG] [handsome](https://osu.ppy.sh/users/2123087)
+  - ![][o!s] [Lexurus - Gravity](https://osu.ppy.sh/beatmapsets/789529#osu/1656879) hosted by ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994)
+  - ![][o!s] [Phonetic - Journey](https://osu.ppy.sh/beatmapsets/795432#osu/1670491) hosted by ![][flag_KR] [Left](https://osu.ppy.sh/users/5062061)
+  - ![][o!s] [Pulse - LP](https://osu.ppy.sh/beatmapsets/789374#osu/1656604) hosted by ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418)
+
+### November 2018
+
+- **[Culprate](https://osu.ppy.sh/home/news/2018-11-21-new-featured-artist-culprate)** (2018-11-21)
```
[the remark this answers is not in the document]
Comment (Nivalyx) [reply wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md:45]: hmm i don't think this is the case here as there are also some other articles out there such as [Mapping and modding timeline](https://osu.ppy.sh/wiki/en/Mapping_and_Modding_Timeline) which uses the same YY-MM-DD format for convenience tho :(

**Comment by Nivalyx** [comment]
oh and on another note @MilkyIQ i applied all of your suggestions, many thanks as always for checking!! 🤍

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md (hunk 4 above)
Comment (peppy) [reply wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md]: Not standard. just osu!. it's not hard.

Comment (DoryanR) [reply wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md]: > Not standard. just osu!. it's not hard.

The only problem is that `osu!standard` is written as such all over the place, if the writing for the `standard` game mode is indeed that one, we should think about changing it all over the place, on each article.

Comment (Walavouchey) [reply wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md]: The ASC needs that change as well if so.

Comment (Nivalyx) [reply wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md]: ah, yeah, i've adjusted the alt texts accordingly in one of my earlier commits dw ;~;

image

image

as for the asc -- yeah it does seem that the asc needs some updating as it still promotes the use of `osu!standard` despite the fact that `osu!` has been the much preferred term for a long while. will open a separate pr on this in a bit

Comment (DoryanR) [reply wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md]: > as for the asc -- yeah it does seem that the asc needs some updating as it still promotes the use of `osu!standard` despite the fact that `osu!` has been the much preferred term for a long while. will open a separate pr on this in a bit

I'm looking into changing `osu!standard` to `osu!` from everywhere, I'll go update at the same time ASC.

**Review by peppy — APPROVED** [review]
Seems fine now?

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md [hunk 6]
```diff
@@ -0,0 +1,369 @@
+# Featured artist showcase beatmaps
+
+Prior to the announcement of upcoming [featured artists](wiki/Featured_Artists), members of the [Mappers' Guild](/wiki/Mappers_Guild) who have achieved either Tier 1 (or higher) in [Guild rank](/wiki/Mappers_Guild#rewards) or been accepting certain [limited-time mystery quests](wiki/Mappers_Guild#quests) set by the Guild in advance are occasionally invited to create exclusive "showcase beatmaps" by using songs and other materials from osu!'s featured artist backlog that are yet to be released. Once it is time to publicly announce the featured artists in question, these beatmaps will be comprehensively covered in the corresponding [newsposts](https://osu.ppy.sh/home/news) and will occasionally be promoted on the [@osugame Twitter](https://twitter.com/osugame "Twitter") as well.
```
Comment (pishifat) [inline wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md:3]:
```suggestion
Members of the [Mappers' Guild](/wiki/Mappers_Guild) occasionally create exclusive "showcase beatmaps" by using songs and other materials from osu!'s unreleased [featured artists](wiki/Featured_Artists). When a featured artist is publicly announced, these beatmaps are covered in the corresponding [newsposts](https://osu.ppy.sh/home/news) and are often promoted on the [@osugame Twitter](https://twitter.com/osugame).

To participate in featured artist showcase beatmaps, a member of the Mappers' Guild must accept a [limited-time mystery quests](wiki/Mappers_Guild#quests) or be directly invited to create a beatmap after reaching tier 1 (or higher) in [Guild rank](/wiki/Mappers_Guild#rewards).
```
no problem with the info, but the way the first sentence is worded is really hard to read (too many clauses)

File: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md [hunk 7]
```diff
@@ -0,0 +1,369 @@
...
+  - (![][osu!]) [fiend - FEVER DREAM (feat. yzzyx)](https://osu.ppy.sh/beatmapsets/1405913#osu/2899195) hosted by ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805)
+
+## History
+
+Even from its very beginning, the Mappers' Guild has been set into pace with "creating promotional beatmap packs for upcoming featured artists" <!-- https://osu.ppy.sh/community/forums/topics/875743 --> as one of its primary goals; in fact, the six beatmaps that were made for the [MOtOLOiD featured artist announcement](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) were also the Guild's first ever creations. 
+
+As the Guild itself was operating on a closed, application-based entry in its early days, access to upcoming featured artist contents were pften restricted to only a handful of mappers who were predominantly of the [osu!](wiki/Game_mode/osu!) game mode; however, the Guild's [shift to an open and automated entry system in early 2019](wiki/Mapping_and_Modding_Timeline#march.5) gradually allowed more and more creators to participate in the creation of showcase beatmaps for future featured artists over time regardless of their game mode.
```
Comment (pishifat) [inline wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md:332]:
```suggestion
As the Guild itself was operating on a closed, application-based entry in its early days, access to upcoming featured artist contents were often restricted to only a handful of mappers who were predominantly of the [osu!](wiki/Game_mode/osu!) game mode; however, the Guild's [shift to an open and automated entry system in early 2019](wiki/Mapping_and_Modding_Timeline#march.5) gradually allowed more and more creators to participate in the creation of showcase beatmaps for future featured artists over time regardless of their game mode.
```

**Review by pishifat — CHANGES_REQUESTED** [review]

**Comment by Nivalyx** [comment]
didn't notice the typo earlier whoops. tyty @pishifat 🌻

also applied the other suggestion that you provided, hopefully everything should be good to go by now!

**Review by pishifat — APPROVED** [review]

**Pull request merged**